lib/stat_tracker.rb

```diff
@@ -95,6 +95,15 @@ def goals_per_game_per_team
       result[game_team.team_id] << game_team.goals
       result
     end
```

Comment:
Your merge caused one `end` to disappear here!

Reply:
Thank you, I've fixed this too